arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -0,0 +1,916 @@
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Error": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string"
+        },
+        "message": {
+          "type": "string"
+        },
+        "target": {
+          "type": "string"
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorDetail"
+          }
+        }
+      }
+    },
+    "ErrorDetail": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string"
+        },
+        "message": {
+          "type": "string"
+        },
+        "target": {
+          "type": "string"
+        }
+      }
+    },
+    "WorkspaceCollectionList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/WorkspaceCollection"
+          }
+        }
+      }
+    },
+    "WorkspaceList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Workspace"
+          }
+        }
+      }
+    },
+    "OperationList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Operation"
+          }
+        }
+      }
+    },
+    "AzureSku": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "SKU name",
+          "type": "string",
+          "enum": [
+            "S1"
+          ]
+        },
+        "tier": {
+          "description": "SKU tier",
+          "type": "string",
+          "enum": [
+            "Standard"
+          ]
+        }
+      }
+    },
+    "WorkspaceCollectionAccessKeys": {
+      "type": "object",
+      "properties": {
+        "key1": {
+          "description": "Access key 1",
+          "type": "string"
+        },
+        "key2": {
+          "description": "Access key 2",
+          "type": "string"
+        }
+      }
+    },
+    "WorkspaceCollectionAccessKey": {
+      "type": "object",
+      "properties": {
+        "keyName": {
+          "description": "Key name",
+          "type": "string",
+          "enum": [
+            "key1",
+            "key2"
+          ]
+        }
+      }
+    },
+    "Workspace": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Workspace id",
+          "type": "string"
+        },
+        "name": {
+          "description": "Workspace name",
+          "type": "string"
+        },
+        "type": {
+          "description": "Resource type",
+          "type": "string"
```

Comment:
IS there are a definitive set of Resource types? If so, then would you want this to be an enum?